src/assets/images/AlertTriangle.svg

```diff
@@ -1,3 +1,3 @@
 <svg viewBox="0 0 16 16" fill="current" xmlns="http://www.w3.org/2000/svg">
-<path d="M7.99976 6.00015V8.66682M7.99976 11.3335H8.00642M7.07663 2.59464L1.59338 12.0657C1.28924 12.591 1.13717 12.8537 1.15965 13.0693C1.17925 13.2573 1.27776 13.4282 1.43067 13.5394C1.60597 13.6668 1.90948 13.6668 2.5165 13.6668H13.483C14.09 13.6668 14.3935 13.6668 14.5688 13.5394C14.7217 13.4282 14.8203 13.2573 14.8399 13.0693C14.8623 12.8537 14.7103 12.591 14.4061 12.0657L8.92288 2.59463C8.61983 2.07119 8.46831 1.80947 8.27062 1.72157C8.09818 1.64489 7.90133 1.64489 7.72889 1.72157C7.5312 1.80947 7.37968 2.07119 7.07663 2.59464Z" stroke="current" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M7.99976 6.00015V8.66682M7.99976 11.3335H8.00642M7.07663 2.59464L1.59338 12.0657C1.28924 12.591 1.13717 12.8537 1.15965 13.0693C1.17925 13.2573 1.27776 13.4282 1.43067 13.5394C1.60597 13.6668 1.90948 13.6668 2.5165 13.6668H13.483C14.09 13.6668 14.3935 13.6668 14.5688 13.5394C14.7217 13.4282 14.8203 13.2573 14.8399 13.0693C14.8623 12.8537 14.7103 12.591 14.4061 12.0657L8.92288 2.59463C8.61983 2.07119 8.46831 1.80947 8.27062 1.72157C8.09818 1.64489 7.90133 1.64489 7.72889 1.72157C7.5312 1.80947 7.37968 2.07119 7.07663 2.59464Z" stroke="currentColor" stroke-width="1.2" stroke-linecap="round" stroke-linejoin="round"/>

```

Comment:
I thought we decided on `stroke="current"`

Reply:
Also do we want 1.2 to be the stroke width? It won't be pixel perfect if we have a fraction here right?

Reply:
The current only is not working, some icons, if I change to current, everything turns black or disappears, the color is only applied with the currentColor.

Reply:
This icon naturally already had a stroke, but it was too thick compared to the others on Alert. I made it a little thinner to maintain consistency.